sale_order_type/__manifest__.py

```diff
@@ -8,13 +8,14 @@
 
 {
     "name": "Sale Order Type",
-    "version": "11.0.1.3.0",
+    "version": "12.0.1.0.0",
     "category": "Sales Management",
     "author": "Grupo Vermon,"
               "AvanzOSC,"
               "Tecnativa,"
               "Agile Business Group,"
               "Niboo,"
+              "Pesol,"
```

Comment:
This simple migration doesn't grant you co-authorship.

Reply:
Sorry, I will update it

Reply:
OK, done @pedrobaeza